src/qaqc/remove_flag_method.R

```diff
@@ -48,81 +46,77 @@ alter_verification_column <- function(df, dt, choice_arg = choice) {
 # there will need to be some sort of imap so that we can verify that the weekly data and the daily data are the same
 verify_flag_data <- function(weekly_plot_object, daily_plot_object) {
 
-  # filter the data from the plot object to get any points where the
-  # verification column is NA
+  # get data from daily plot
   df_data <- ggplot_build(daily_plot_object)$plot$data
-  flag_data <- df_data %>%
-    filter(is.na(verification))
-
-  # pull the site-param combo and inform the user what they are working on
-  flag_site <- flag_data$site[1]
-  flag_parameter <- flag_data$parameter[1]
+  df_name <- ggplot_build(daily_plot_object)$plot$labels$title
 
   # which df are you working on:
-  cat("Currently verifying: ", flag_site, flag_parameter, " data.\n")
+  cat("Currently verifying: ", df_name, " data.\n")
+
+  print(ggarrange(weekly_plot_object, daily_plot_object, nrow = 2, ncol = 1))
+  # Generate the prompt for the user to answer
+  day_choice_prompt <- paste("Would you like to (pass/fail/inspect) all data points for: ", df_name, "? (p/f/i): ")
+  day_choice <- get_day_decision(day_choice_prompt)
+
+  # alter the df based on the response from the user
+  if (day_choice == "pass") { # if choice is true
+    # add pass to verification column for all rows
+    altered_df <- df_data %>%
+      mutate(verification = "pass")
+    cat("All points for ", df_name, "have PASSED.\n")
+    return(altered_df)
+  } else if (day_choice == "fail") { # if choice is false
+    # add fail to verification column for all rows
+    altered_df <- df_data %>%
+      mutate(verification = "fail")
+    cat("All points for ", df_name, "have FAILED.\n")
+    return(altered_df)
+  } else if (day_choice == "inspect") {
+
+    # pull the flagged data and iterate through each row
+    flag_data <- df_data %>%
+      filter(is.na(verification))
+
+    for (i in 1:nrow(flag_data)){
+      # Create a data frame with a single row containing the desired DT_round value
+      # and add it to the week plot and day plot
+      flag_demarcation_data <- data.frame(DT_round = flag_data$DT_round[i])
+
+      week_plot <- weekly_plot_object +
+        geom_vline(data = flag_demarcation_data,
+                   aes(xintercept = DT_round,
+                       color = "data point of interest",
+                       # size = 2, # fix this later
+                       alpha = 0.05))
+
+      day_plot <- daily_plot_object +
+        geom_vline(data = flag_demarcation_data,
+                   aes(xintercept = DT_round,
+                       color = "data point of interest",
+                       # size = 2, # fix this later
+                       alpha = 0.05))
+
+      # Print the plot so the user can see the data
+      print(ggarrange(week_plot, day_plot, nrow = 2, ncol = 1))
+
+      # Generate the prompt for the user to answer
+      flag_dt <- as.character(flag_data$DT_round[i])
+      flag_choice_prompt <- paste(flag_dt, " (pass/fail): ")
```

Comment:
This seems like a lot of prompting... 15 min data for 1 day is like 100 data points. Can we make a prompt where you can select a few hours instead?
I realize that would be a bit complicated but reviewing 100 data points per sensor per site would hurt my brain...